vignettes/get_data_inboveg.Rmd

```diff
@@ -77,10 +77,10 @@ survey_info <- get_inboveg_survey(con,
 
 ```{r}
 survey_info
-#> # A tibble: 1 x 5
-#>      Id Name        Description                                   Owner creator 
-#>   <int> <chr>       <chr>                                         <chr> <chr>   
-#> 1   172 OudeLanden~ Verlinden A, Leys G en Slembrouck J (1979) G~ NA    els_deb~
+# A tibble: 1 x 5
+#      Id Name        Description                                   Owner creator 
+#   <int> <chr>       <chr>                                         <chr> <chr>   
+#  1   172 OudeLanden~ Verlinden A, Leys G en Slembrouck J (1979) G~ NA    els_deb~
```

Comment:
@ElsDeBie I would just remove all such lines (the commented out output) from the r-chunks (not only here, in all chunks). @ElsLommelen will work out a way that the vignette can be build locally and does not give problems with package checks.

Reply:
And if you wish to show the output, you can already remove the `eval = FALSE` in the first chunk.  This means we have to build the website locally when the vignette has changed, but this will stay the case with the option suggested by @hansvancalster.

Reply:
I now notice that sometimes there is a code to show the result, and sometimes there is just a code without showing the result. What is the best to do? for example
```{r examples_survey}
survey_info <- get_inboveg_survey(con, 
                              survey_name = "OudeLanden_1979", 
                              collect = TRUE)
```
```{r}
survey_info
```
so the result is shown, (isn't it beter to put both in one chunck?)

or 
```{r}
header_severalsurveys <- get_inboveg_header(con,
                survey_name = c("MILKLIM_Heischraal2012", "NICHE Vlaanderen"),
                multiple = TRUE)
```
and then a next example is shown without given the result of "header_severalsurveys"

Reply:
It depends on the purpose and what you are explaining/illustrating, I think.  If the focus in the text is on the output, it is definitively necessary to show the output.  If you just want to illustrate some different coding possibilities (all of them having more or less the same output), it may not always be necessary to show the full output for each of them.

So I would check in the text what seems appropriate to you, and keep in mind that users may read this manual before deciding on using the package.  (Different from the examples at the help functions, this is a document to read, so users are probably less likely to try the code, and expect the results to be shown whenever appropriate.)